### ISSUE TITLE
Review notify UI/UX for normal users and for screen reader users

### ISSUE THREAD

**Issue by jardakotesovec** [pkp/pkp-lib#9696]
We have announcer for SR users as well as 'notify' event on global bus to display notifications in top right corner. 

Review guidelines on which occasions these should be used.

Also make sure API unexpected error responses are always presented same way. Currently some of them are presented via dialog and some via notifications.

TODO: collect some examples here.
